.gitignore

```diff
@@ -1,5 +1,9 @@
 conftest
 dist
+<<<<<<< HEAD
```

Comment:
@Salamandastron1 
there is an unresolved merge conflict in the .gitignore file. 
lets clean this up before any PR merge :)